src/main.d

```diff
@@ -332,6 +357,11 @@ int main(string[] args)
 						log.vdebug("updating sync_list hash as --resync issued");
 						std.file.write(syncListHashFile, computeQuickXorHash(buildNormalizedPath(cfg.configDirName ~ "/sync_list")));
 					}
+					if (exists(buildNormalizedPath(cfg.configDirName ~ "/business_shared_folders"))) {
```

Comment:
Again normalizing the same

Reply:
Hrmm ... will check .. forest / trees and the like!

Reply:
Once #961 is merged into 'master', that will update most of these, then will update the remaining from this PR

Reply:
https://github.com/abraunegg/onedrive/pull/473/commits/2304df10770de3e6401ddcf485105fdde54d5c7f fixes this